lib/Certificate.php

```diff
@@ -140,6 +143,26 @@ public function isSelfSigned()
         return $this->info["subject"] === $this->info["issuer"];
     }
 
+    public function getKeySha256Base64(): string
+    {
+        return \base64_encode($this->getKeySha256Raw());
+    }
+
+    public function getKeySha256Hex(): string
+    {
+        return \bin2hex($this->getKeySha256Raw());
+    }
+
+    public function getKeySha256Raw(): string
+    {
+        $publicKeyPem = \openssl_pkey_get_details(\openssl_pkey_get_public($this->resource))['key'];
+
+        $publicKeyBase64 = \array_slice(\explode("\n", $publicKeyPem), 1, -2);
+        $publicKeyRaw = \base64_decode(\implode('', $publicKeyBase64));
+
+        return \hash('sha256', $publicKeyRaw, true);
```

Comment:
`false` would be more useful I think - ppl can always decode if they need the raw version - yet most should just compare the b64 one and be done.

Reply:
For the base64 one, it's better to have the raw hash and then base64 encode it, instead of hex encoding, then hex decoding and then base64 encoding.